jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -250,15 +250,21 @@ def mos_image_parser(app, data_obj, data_labels=None):
         the mosviz table.
     data_labels : str, optional
         The label applied to the glue data component.
+    share_image : int, optional
+        If 0, images are treated as applying to individual spectra. If non-zero,
```

Comment:
As an outsider, it is still unclear to me. What is the difference if I pass in `share_image=1` and `share_image=2`?

Reply:
As a user, you don't pass in that argument at all. The user calls one of the `load_` methods, which in turn call the parsers on the backend. This is an argument that gets populated automatically based on the input to the `load_data` method, so the docstring is really developer-facing IMO.

Reply:
Still, I guess you're a developer, so it should be useful to you...

Reply:
FWIW, I also was confused what the meaning of "not 0" is from the docstring.